ui/src/App.js

```diff
@@ -23,6 +23,9 @@ class App extends Component {
     var apiClient = new ApiClient();
     apiClient.basePath = '/api'
     var api = new FacetsApi(apiClient)
+    var opts = {
```

Comment:
I'm assuming the changes to this file were for testing purposes? If so, can we revert?

Reply:
The change to line 37 was necessary. Without that change, UI doesn't work. Facets response is not printed in console.

Reply:
Then can we just keep line 37 as`api.facetsGet([], callback)`? This function is the initial population of Facets, which should return all of them.

Reply:
Done. Just be aware that ui/src/api/README.md has sample code.